.github/workflows/karate.yml

```diff
@@ -63,5 +63,5 @@ jobs:
       - name: Delete karate test users
         run: BASE_PATH=${{env.BASE_PATH}} ./gradlew deleteTestUsers --info
         # Delete stack after running tests
-      - name: Delete stack after running tests
-        run: sam delete --no-prompts --region eu-west-1 --stack-name ${{env.STACK_NAME}}
+      #- name: Delete stack after running tests
+        #run: sam delete --no-prompts --region eu-west-1 --stack-name ${{env.STACK_NAME}}
```

Comment:
?

Reply:
it will be reverted before merge. just a way to do a feature branch test for frontend